plugins/topology/src/utils/pod-resource-utils.ts

```diff
@@ -0,0 +1,280 @@
+import * as _ from 'lodash';
```

Comment:
I suggest importing the specific resources which is utilized in the source code.

```suggestion
import { get, isFinite, some } from 'lodash';
```